src/table/boot.rs

```diff
@@ -480,6 +484,18 @@ impl BootServices {
             (self.start_image)(image_handle, &mut exit_data_size, &mut exit_data).into()
         }
     }
+    
+    /// Exits the UEFI application and returns control to the UEFI component
```

Comment:
Should we also mention here that before exiting, the application is responsible for freeing any resources it allocated as defined in the UEFI specs?

Reply:
Resolved in next commit.